src/main/java/lotto/domain/WinningLotto.java

```diff
@@ -0,0 +1,46 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class WinningLotto {
+    private Lotto winningLotto;
+    private LottoNumber bonusNumber;
+
+    public WinningLotto(Lotto inputWinningLotto, LottoNumber inputBonusNumber) {
+        validateBonusNumber(inputWinningLotto, inputBonusNumber);
+        this.winningLotto = Lotto.manual(inputWinningLotto.getLottoNumbers());
+        this.bonusNumber = LottoNumber.chooseNumber(inputBonusNumber.getLottoNumber());
+    }
+
+    public WinningLotto(Lotto inputWinningLotto, int inputBonusNumber) {
+        this(inputWinningLotto, LottoNumber.chooseNumber(inputBonusNumber));
+    }
+
+    public WinningLotto(List<Integer> inputWinningLottoNumbers, LottoNumber inputBonusNumber) {
+        this(Lotto.manual(inputWinningLottoNumbers), inputBonusNumber);
+    }
+
+    public WinningLotto(List<Integer> inputWinningLottoNumbers, int inputBonusNumber) {
+        this(Lotto.manual(inputWinningLottoNumbers), LottoNumber.chooseNumber(inputBonusNumber));
+    }
+
+    public Lotto getWinningLotto() {
+        return Lotto.manual(this.winningLotto.getLottoNumbers());
+    }
+
+    public LottoNumber getBonusNumber() {
+        return this.bonusNumber;
+    }
+
+    public RankEnum getRank(Lotto lotto) {
+        int matchedCount = getWinningLotto().getMatchedCount(lotto);
+        boolean hasBonus = lotto.isExistNumber(getBonusNumber());
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/java-lotto/pull/438#discussion_r402377995

```suggestion
        int matchedCount = this.winningLotto.getMatchedCount(lotto);
        boolean hasBonus = lotto.isExistNumber(this.bonusNumber);
```